x/slashing/client/rest/query.go

```diff
@@ -2,44 +2,84 @@ package rest
 
 import (
 	"fmt"
-	"net/http"
-
-	"github.com/gorilla/mux"
-
 	"github.com/cosmos/cosmos-sdk/client/context"
 	"github.com/cosmos/cosmos-sdk/codec"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/cosmos/cosmos-sdk/types/rest"
 	"github.com/cosmos/cosmos-sdk/x/slashing"
+	"github.com/gorilla/mux"
+	"net/http"
 )
 
 func registerQueryRoutes(cliCtx context.CLIContext, r *mux.Router, cdc *codec.Codec) {
 	r.HandleFunc(
-		"/slashing/validators/{validatorPubKey}/signing_info",
+		"/slashing/validators/signing_info/{validatorPubKey}",
 		signingInfoHandlerFn(cliCtx, slashing.StoreKey, cdc),
 	).Methods("GET")
 
+	r.HandleFunc(
+		"/slashing/validators/signing_info",
+		signingInfoHandlerListFn(cliCtx, slashing.StoreKey, cdc),
+	).
```

Comment:
This formatting/indentation seems very off?

Reply:
I didn't know how to chain Methods and Queries at the same indentation, there is an alternative to what I wrote? I found having all in the same line was less readable

Reply:
Let's just do:

```
r.HandleFunc(
  // ...
).Methods("GET").Queries("page", "{page}", "limit", "{limit}")
```